test/e2e/run_networking_test.go

```diff
@@ -1128,40 +1129,8 @@ EXPOSE 2004-2005/tcp`, ALPINE)
 		session = podmanTest.Podman([]string{"run", "--name", "con4", "--network", net, ALPINE, "nslookup", pod2 + ".dns.podman"})
 		session.WaitWithDefaultTimeout()
 		Expect(session).Should(Exit(0))
-	})
-
-	It("podman run check dnsname plugin with Netavark", func() {
```

Comment:
Nice catch. I took a moment just now to copypaste each of those tests into tmpfiles, then diff them. They are 100% identical other than test name and the (useless) `SkipIfCNI()`.